docs/developers/reference/api/linea-gettransactionexclusionstatusv1.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: linea_getTransactionExclusionStatusV1
+description: Reference content for the linea_getTransactionExclusionStatusV1 method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getTransactionExclusionStatusV1`
+
+Queries a temporary database maintained by the transaction exclusion API service to check if a 
+was rejected for exceeding data line limits that would prevent the prover from generating a 
+proof for the block. The database is updated by a parallel write API endpoint called by the 
+sequencer, edge nodes, or RPC nodes.
+
+If the transaction was rejected, the API call will be successful and provide the reason. 
+
+Transactions that were not rejected for this reason will return `null` as the `result`. 
+
+:::warning
+Transactions older than 24 hours can't be checked.
```

Comment:
```suggestion
Transaction rejection reason can only be checked within 24 hours of transaction attempt.
```

Reply:
`within 24 hours of transaction attempt` doesn't sound equivalent to the actual behavior which is something like `within 24 hours since transaction was rejected`, any better wording suggestion here?

Reply:
I think you did it with that !